docs/en/operations/server-configuration-parameters/settings.md

```diff
@@ -746,16 +746,17 @@ Path to temporary data for processing large queries.
 <tmp_path>/var/lib/clickhouse/tmp/</tmp_path>
 ```
 
-## tmp\_policy {#server-settings-tmp-policy}
+## tmp_policy {#tmp-policy}
```

Comment:
manually removing underscore escaping is not really useful, it might get overwritten back at any moment